src/foremast/runner.py

```diff
@@ -170,6 +171,8 @@ def create_dns(self):
             dnsobj.create_failover_dns(primary_region=primary_region)
         else:
             dnsobj.create_elb_dns(regionspecific=False)
+            if regionspecific_dns: #If true, Also create a region specific DNS record
+                dnsobj.create_elb_dns(regionspecific=True)
```

Comment:
Why does it matter? It is creating `app.group.region.env.domain.com` and `app.group.env.domain.com`. Both of which are CNAMEs to the same ELB record.